src/osemgrep/configuring/Semgrep_envvars.ml

```diff
@@ -54,15 +54,19 @@ let in_env var = env_opt var <> None
 type t = {
   semgrep_url : Uri.t;
   fail_open_url : Uri.t;
+  metrics_url : Uri.t;
   app_token : string option;
+  integration_name : string option;
   version_check_url : Uri.t;
   version_check_timeout : int;
   version_check_cache_path : Fpath.t;
   git_command_timeout : int;
   src_directory : Fpath.t;
+  user_agent_append : string option;
```

Comment:
maybe a comment explaining in which context this environment variable is useful.
Who is using it?

Reply:
Sure. It's currently used in our Dockerfile image.

Reply:
added docstrings in the `.mli`